hw_5.py

```diff
@@ -0,0 +1,9 @@
+﻿"""
+Write a program to read last n lines of a file.
+"""
+
+
+def read_last_N_lines():
```

Comment:
It's a bad idea.
You've hardcoded the number.

The common way is to have `n` as a parameter.

Reply:
done